packages/vite/src/node/http.ts

```diff
@@ -2,13 +2,79 @@ import fs, { promises as fsp } from 'fs'
 import path from 'path'
 import { Server as HttpServer } from 'http'
 import { ServerOptions as HttpsServerOptions } from 'https'
-import { ResolvedConfig, ServerOptions } from '..'
-import { isObject } from '../utils'
+import { isObject } from './utils'
+import { ProxyOptions } from './server/middlewares/proxy'
 import { Connect } from 'types/connect'
-import { Logger } from '../logger'
+import { Logger } from './logger'
+
+export interface CommonServerOptions {
+  port?: number
```

Comment:
All other properties have JSDoc, so I think we should also add it for `port` and `host`